performance-tests/src/libs/backbone-client/identity.ts

```diff
@@ -1,10 +1,10 @@
+import { Httpx } from "https://jslib.k6.io/httpx/0.1.0/index.js";
 import { b64encode } from "k6/encoding";
 import { Response } from "k6/http";
-import { ChallengeResponse } from "../../models/challenge";
-import { CreateIdentityRequest, CreateIdentityResponse } from "../../models/identity";
-import { TokenResponse } from "../../models/token";
-import { ChallengeRequestRepresentation, CryptoHelper } from "../crypto-helper";
-import { Httpx } from "https://jslib.k6.io/httpx/0.1.0/index.js";
+import { CreateChallengeResponse } from "../../models/challenge";
+import { CreateIdentityRequest } from "../../models/identity";
+import { JwtResponse } from "../../models/jwt-response";
+import { ChallengeRequestPayload, CryptoHelper } from "../crypto-helper";
 
 export function createIdentity(client: Httpx, clientId: string, clientSecret: string): { httpResponse: Response; generatedPassword: string } {
```

Comment:
If we need the password outside of this method, we should probably rather pass it as a parameter into this method, instead of returning it to the caller.